guides/v2.2/cloud/project/project-upgrade.md

```diff
@@ -1,259 +1,199 @@
+-  Upgrade your PHP version to 7.0 or later
+-  Create a new `config.php` file
+-  Update the `.magento.app.yaml` file with new settings for hooks and environment variables
+-  Verify or set the `ADMIN_EMAIL` variable
+-  Upgrade to the latest supported version of Fastly
+-  Update the `.gitignore` file for new, generated directory
 
-You can attempt to upgrade from any version directly to {{site.data.var.ece}} 2.2. We cannot guarantee the results. For example, you should be able to upgrade from 2.0.10 or 2.1.4 directly to 2.2.
+{% include cloud/note-upgrade.md %}
 
-If you prefer a secured and verified upgrade path, you can upgrade to one of the verified and tested versions, then directly upgrade to 2.2. For example, you could upgrade from 2.0.10 to 2.0.14, then upgrade to 2.2.
+{% include cloud/note-ece-tools-package.md %}
 
-When upgrading from any version to 2.2, please review the following sections to update your settings, make changes, and upgrade required software prior to upgrading Magento.
-
-## Upgrade from 2.0.X or 2.1.X {#old-version}
-To upgrade from **2.0.X**:
-
-* [Upgrade your PHP version](#php): v2.2 supports PHP 7.0 and later
-* [.magento.app.yaml](#magento-app-yaml): Update the file with new settings and required changes for hooks and environment variables
-* [Verify or set the ADMIN_EMAIL variable](#variable): This variable is required for upgrades and patch to 2.2 and later
-* [Upgrade Fastly](#fastly): Make sure you are upgraded to the latest supported version of Fastly
-
-To upgrade from **2.1.X**:
-
-* [Upgrade your PHP version](#php): v2.2 supports PHP 7.0 and 7.1
-* [Configuration Management](#config): Create a new `config.php` using the `config.local.php` to properly upgrade
-* [.magento.app.yaml](#magento-app-yaml): Update the file with new settings and required changes for hooks and environment variables
-* [Verify or set the ADMIN_EMAIL variable](#variable): This variable is required for upgrades and patch to 2.2 and later
-* [Upgrade Fastly](#fastly): Make sure you are upgraded to the latest supported version of Fastly
-* [Update .gitignore for new generated directory](#gitignore): With 2.2 and later, changes in generated folder require additions to `.gitignore`
-
-After completing your upgrade, you may also want to [Verify and upgrade your extensions](#extensions).
-
-### Upgrade your PHP version {#php}
-{{site.data.var.ece}} 2.2 supports PHP 7.0 and 7.1. For Pro projects **created before October 23, 2017**, you must open a [support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to use PHP 7.1 on your Pro Staging and Production environments.
-
-Make sure to upgrade the version of PHP on your local development workspace as well. For more information, see the following:
+### Upgrade PHP version
+{{site.data.var.ece}} 2.2 supports PHP 7.0 and 7.1. Make sure to upgrade the version of PHP on your local development workspace as well. For more information, see the following:
```

Comment:
Is it a local development "workspace" or "workstation"? I've seen references to both.